### PR TITLE
Kris

### PR DESCRIPTION
**Pull request by kadamowi**
Sorry for too many commits, I'm still learning
This is new metric which check if Kontakt page exists

### CODE REVIEW

**Comment by bartoszwalacik** [comment]
please try giving more meaningful names for branches (and PR)

File: .idea/encodings.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bartoszwalacik) [inline .idea/encodings.xml]: we do not commit local dot files to scm,

remove it and add to `.gitignore`
```
# JetBrains
.idea/
*.iml
*.ipr
*.iws
```

File: app/analysis/contact.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+module.exports = {
+    name: 'contact',
+    description: 'Czy na stronie jest odnosnik do strony z informacją kontaktowa ?',
+    run: function (page, callback) {
+        page.evaluate(function () {
+            var allHypers = document.querySelectorAll('a');
+            var i;
+            var hrefText = '';
+            for (i = 0; i < allHypers.length; i++) {
+              if (allHypers[i].innerText === 'Kontakt' && allHypers[i].href !== '') {
```
Comment (bartoszwalacik) [inline app/analysis/contact.js]: a jeśli kontakt będzie z małej?

File: app/analysis/cms.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+module.exports = {
+    name: 'cms',
+    description: 'Czy strona jest budowana przy pomocy narzedzie CMS ?',
+    run: function (page, callback) {
+        page.evaluate(function () {
+            var allMeta = document.querySelectorAll('meta[name="generator"]');
+            var cmsName;
+            var n;
+            if (allMeta.length === 1) {
```
Comment (bartoszwalacik) [inline app/analysis/cms.js]: lot of nested if's, it looks scary ..
why not use array with cms names?

btw, we need test for all cases (now, there is test only for Joomla)

**Comment by kadamowi** [comment]
Ok, from next PR

**Pull request merged**